Content.Client/Info/RulesManager.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using Robust.Shared.ContentPack;
+using Robust.Shared.IoC;
+using Robust.Shared.Network;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Info;
+
+public sealed class RulesManager
+{
+    [Dependency] private readonly IClientNetManager _clientNetManager = default!;
+
+    // If you fork SS14, change this to have the rules "last seen" date stored separately.
+    public const string ForkId = "Wizards";
```

Comment:
Isn't there a better way to get Fork ID? We should have this in a unified location somewhere.

Reply:
Make this a CVar.

Reply:
This is a copy of how changelog does it. It should probably be changed to some other ID like server ID so that rules aren't tied to the codebase, since I think some code bases have different rules on each server even if they have the same code.

Should be fine like this for now though, right?

Reply:
Ah yeah, a CVar makes more sense.